src/tacacs/audisp/Makefile

```diff
@@ -0,0 +1,30 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = audisp-tacplus_$(AUDISP_TACPLUS_VERSION)_$(CONFIGURED_ARCH).deb
+DERIVED_TARGETS = libtac2_$(PAM_TACPLUS_VERSION)_$(CONFIGURED_ARCH).deb \
+		  libtac-dev_$(PAM_TACPLUS_VERSION)_$(CONFIGURED_ARCH).deb
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Obtain audisp-tacplus
+	rm -rf ./audisp-tacplus
+
+	git clone https://github.com/daveolson53/audisp-tacplus.git
+
+	# checkout by sha1
+	pushd ./pam_tacplus
+	git checkout 559c9f22edd4f2dea0ecedffb3ad9502b12a75b6
```

Comment:
>559c9f22edd4f2dea0ecedffb3ad9502b12a75b6 [](http://example.com/codeflow?start=14&length=40)

How do you choose this commit? Is it on specific branch?

Reply:
This commit is the latest commit on master branch.
Our patch will base on this commit.